metadata.json

```diff
@@ -6,7 +6,7 @@
   "source": "git://github.com/antoineco/aco-ulyaoth.git",
   "summary": "Puppet module for Ulyaoth YUM repository",
   "version": "2.0.0",
-  "tags": ["yum","tomcat","nginx","logstash","kibana","rpm","repository"],
+  "tags": ["yum","red hat","amazon linux","fedora","oracle linux","scientific linux","centos","rpm","repository","ulyaoth"],
```

Comment:
The tags you added are discouraged because they are redundant with:
* the `operatingsystem_support` metadata
* the module name

I'd rather not add them. A maximum of 4 tags which describe accurately what the module manages (in this case a Yum repo) is recommended.